### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by Xaeroxe**
As @torkleyy requested I reviewed the docs for specs.

I found a few sores that I've corrected.  Let me know if you can think of anything that I missed.

By the way the specs::bitset module was removed from the documentation because it's doc page was useless and had only two sentences on it.

---
This change is [Reviewable](https://reviewable.io/reviews/slide-rs/specs/378)

### CODE REVIEW

File: src/bitset.rs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#![doc(hidden)]
```
Comment (torkleyy) [inline src/bitset.rs]: Just remove pub from the module maybe?

File: book/src/06_system_data.md [hunk 2]
```diff
@@ -18,8 +18,8 @@ impl<'a> System<'a> for Sys {
 }
 ```
 
-It is very important that you don't fetch both a `ReadStorage` and a `WriteStorage`
-for the same component or a `Fetch` and a `FetchMut` for the same resource.
+It is very important that you don't Read both a `ReadStorage` and a `WriteStorage`
```
Comment (WaDelma) [inline book/src/06_system_data.md]: First of all read should either be de-capitalized or made code.
After that the sentence is super awkward: It's implying that there is reading of a `WriteStorage`.
Not sure how to fix it.

File: book/src/07_join.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Joining components
 
 In the last chapter, we learned how to access resources using `SystemData`.
-To access our components with it, we can just fetch a `ReadStorage` and use
+To access our components with it, we can just get a `ReadStorage` and use
 `Storage::get` to get the component associated to an entity. This works quite
```
Comment (WaDelma) [inline book/src/07_join.md]: It's kinda unfortunate that there is three different uses of 'get' here, but it's not that big of a problem.

File: src/lib.rs [hunk 4]
```diff
@@ -214,7 +214,7 @@ pub mod saveload;
 /// Implementations and structures related to bitsets.
 ///
 /// Normally used for `Join`s and filtering entities.
-pub mod bitset;
+mod bitset;
```
Comment (WaDelma) [inline src/lib.rs]: Did you verify that there is no uses for this? (In benchmarks and examples as those are not automatically tested afaik).

File: src/saveload/mod.rs [hunk 5]
```diff
@@ -4,25 +4,25 @@
...
 //!
-//! Fetching those makes it very easy to serialize or deserialize
-//! components. However, be aware that you cannot fetch storages
+//! Reading those makes it very easy to serialize or deserialize
+//! components. However, be aware that you cannot read storages
 //! used in `WorldDeserialize` with the same system. E.g.
 //! `type SystemData = (WorldDeserialize<'a, Marker, MyError, (Pos, Vel)>, WriteStorage<'a, Vel>)`
-//! is not valid since both `WorldDeserialize` and `WriteStorage` would fetch the same component
+//! is not valid since both `DeserializeComponents` and `WriteStorage` would fetch the same component
 //! storage mutably.
```
Comment (WaDelma) [inline src/saveload/mod.rs]: Is this documentation up to date on saveload stuff?

File: src/lib.rs (hunk 4 above)
Comment (Xaeroxe) [reply src/lib.rs]: @WaDelma There's no way that there could be usages.  This module only contains trait implementations and nothing else.  The docs for it don't generate anything except for the doc comments.

File: src/saveload/mod.rs (hunk 5 above)
Comment (Xaeroxe) [reply src/saveload/mod.rs]: @WaDelma Yes, do you have reason to believe otherwise?

Comment (WaDelma) [reply src/saveload/mod.rs]: Ah, so there are just renamings missing?
```
 //! used in `WorldDeserialize` with the same system. E.g.
 //! `type SystemData = (WorldDeserialize<'a, Marker, MyError, (Pos, Vel)>, WriteStorage<'a, Vel>)`
```

Also the the saveload example looks totally different.

Comment (torkleyy) [reply src/saveload/mod.rs]: I think World* does not exist anymore.

Comment (torkleyy) [reply src/saveload/mod.rs]: I wanted to eventually add them back but not sure now and don't have much time, I'll let you two decide.

**Comment by Rhuagh** [comment]
Once this is finished, I'll go through the book and try to correct anything that's broken, incomplete or outright wrong.

**Comment by Xaeroxe** [comment]
@WaDelma I fixed all the problems you pointed out, thanks for catching those!

**Comment by Rhuagh** [comment]
Ready for merge?

**Pull request merged**